Assets/LeapMotionModules/ElementRenderer/Scripts/SupportInfo.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+
+[Serializable]
+public struct SupportInfo {
```

Comment:
It was originally called that, but Feature is a specific thing in this system, and I wanted to communicate that this class was for more than just supporting features, you could also give renderer support, or space support.